app/assets/stylesheets/components/list-items.scss

```diff
@@ -0,0 +1,21 @@
+////////////////////////////////////////////////////////////
+// .c-list-item ////////////////////////////////////////////
+////////////////////////////////////////////////////////////
+
+.c-list-item {
+  padding: var(--su-3);
+  margin: 0 calc(var(--su-3) * -1);
+  border-bottom: 1px solid var(--divider);
+
+  &:last-child {
+    border-bottom: none;
+  }
+
+  &:not(&--no-interactive) {
+    &:hover,
+    &:focus-visible,
+    &:focus-within {
+      background: var(--list-item-bg-hover);
+    }
+  }
+}
```

Comment:
Draft for a new component - it's been used by design in few places but I think it's still too early to write a documentation for it - we just haven't explored it well-enough just yet.